src/averageAges.js

```diff
@@ -15,13 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = (!century)
+    ? people.filter((human) => human.sex === 'm')
+    : people.filter((human) => (human.sex === 'm'
+      && Math.ceil(human.died / 100) === century));
+
+  const averageAgeMan = averageAge(menArray);
+
+  return averageAgeMan;
 }
 
+function averageAge(array) {
```

Comment:
Rename according to [checklist point 1](https://github.com/mate-academy/js_average-ages/blob/master/checklist.md)